packages/react-scripts/package.json

```diff
@@ -58,7 +58,7 @@
     "fs-extra": "^9.0.1",
     "html-webpack-plugin": "4.5.0",
     "identity-obj-proxy": "3.0.0",
-    "jest": "26.6.0",
+    "jest": "27.0.0-next.6",
     "jest-circus": "26.6.0",
```

Comment:
Probably want to update this as well? And `babel-jest`? Basically anything with `jest` in it that has a new release.

Reply:
I see, with the same (`27.0.0-next.6`) version I presume?

Moreover, running into test failure in `packages/react-error-overlay/src/__tests__/get-source-map.js`. I tried adding a `testEnvironment` docblock but then it throws `global` is not defined.

Reply:
Yes, all Jest package versions should all go in lockstep.

Reply:
>Moreover, running into test failure in packages/react-error-overlay/src/__tests__/get-source-map.js. I tried adding a testEnvironment docblock but then it throws global is not defined.

@SimenB any known breaking changes that cause this? The error says:

```
    The error below may be caused by using the wrong test environment, see https://jestjs.io/docs/en/configuration#testenvironment-string.
    Consider using the "jsdom" test environment.

    ReferenceError: window is not defined
```

Reply:
TIL about the word `lockstep`

Reply:
> @SimenB any known breaking changes that cause this? The error says:

Yep, we've changed default test environment to `node` instead of `jsdom`